boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts

```diff
@@ -5,6 +5,8 @@
  */
 #include "nrf5340_audio_dk_nrf5340_cpuapp_common-pinctrl.dtsi"
 
+#include <zephyr/dt-bindings/sensor/ina230.h>
```

Comment:
This file is called ina230.h, but the board is using INA231.
Maybe change name to something like ina23x.h?

Reply:
upstream driver is compatible with both, but they named it 230. We can modify it upstream to make things more clear, but should not block this PR.